cleo/commands/completions_command.py

```diff
@@ -394,9 +391,9 @@ def get_shell_type(self) -> str:
         return os.path.basename(shell)
 
     def _generate_function_name(self, script_name: str, script_path: str) -> str:
-        return "_{}_{}_complete".format(
-            self._sanitize_for_function_name(script_name),
-            hashlib.md5(script_path.encode()).hexdigest()[0:16],
+        return (
+            f"_{self._sanitize_for_function_name(script_name)}"
+            f"_{hashlib.md5(script_path.encode()).hexdigest()[0:16]}_complete"
```

Comment:
this might be more readable by assigning intermediate variables first and then interpolating them after, something like `return f"_{sanitized_name}_{hash}_complete"`